src/helm/benchmark/static/schema_instruction_following.yaml

```diff
@@ -0,0 +1,238 @@
+---
+############################################################
+models:
```

Comment:
I thought we are reading information out of the metadata yaml now...do we still need this?

Reply:
Removed.